neps/nep-0359.md

```diff
@@ -0,0 +1,168 @@
+
+However, [NEP-178](./nep-0178.md), which pertains to approval management is not applicable for this standard.
+
+### Events
+
+When reporting events in order to conform to the event standard [NEP-297](./nep-0297.md) this standard would be similar to the events specification for NEP-171, however there would be no need for an `nft_transfer` event.
+
+More explicitly then, Non Transferable Non-Fungible Token Events MUST have `standard` set to `"nep359"`, standard version set to `"1.0.0"`, `event` value is one of `nft_mint` or `nft_burn` and `data` must be of one of the following relavant types: `NftMintLog[] | NftBurnLog[]`:
+
+```ts
+interface NftEventLogData {
+    standard: "nep359",
+    version: "1.0.0",
+    event: "nft_mint" | "nft_burn"
+    data: NftMintLog[] | NftBurnLog[],
+}
+```
+
+```ts
+// An event log to capture token minting
+// Arguments
+// * `owner_id`: "account.near"
+// * `token_ids`: ["1", "abc"]
+// * `memo`: optional message
+interface NftMintLog {
+    owner_id: string,
+    token_ids: string[],
+    memo?: string
+}
+
+// An event log to capture token burning
+// Arguments
+// * `owner_id`: owner of tokens to burn
+// * `authorized_id`: approved account_id to burn, if applicable
+// * `token_ids`: ["1","2"]
+// * `memo`: optional message
+interface NftBurnLog {
+    owner_id: string,
+    authorized_id?: string,
+    token_ids: string[],
+    memo?: string
+}
+```
+
+## Reference Implementation
+
+[Minimum Viable Interface](https://github.com/kycdao/near-sdk-rs/blob/ntnft/near-contract-standards/src/ntnft/core/mod.rs)
+
+[NFT Implementation](https://github.com/kycdao/near-sdk-rs/blob/ntnft/near-contract-standards/src/ntnft/core/core_impl.rs)
+
+## Unresolved Issues
+
+### Dealing with minted NTNFTs in cases of lost, stolen or new accounts
+
+Implementers should consider the case of what should happen to these NTNFTs if the user loses access to their account, it is somehow compromised or they simply wish to use a new account. As there are a number of ways this can be handled, this standard does not explicitly mandate that any particular function MUST be implemented to handle this.
+
+For each of these methods, it would presumably be necessary for the service which created the smart contract to make efforts (off-chain) to identify the user. Once they're satisfied the user has identified themselves they could use any of the following methods to invalidate the old NTNFT and, if required, mint a new token (presumably with the same metadata) to the new account.
+
+- The service would burn the NTNFT in the user's old account using: `function burn_token(token_id: string): null {}`. Giving the user access to this function may also be considered - in cases where they still have access to their account users could pre-emptively call this to prevent anyone using their NTNFTs to impersonate them.
```

Comment:
There isn't a standard for burning tokens - should we enforce the name of the function here? I think it would be fine to simply say the token should be burnt and an event should be emitted.
```suggestion
- The service would burn the NTNFT in the user's old account and an `NftBurnLog` should be emitted. Giving the user access to this functionality may also be considered - in cases where they still have access to their account users could pre-emptively do this to prevent anyone using their NTNFTs to impersonate them.
```